content/pages/pricing.html

```diff
@@ -2,389 +2,263 @@
-                    <div class="description">on every commit</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Multi-version handling</div>
-                    <div class="description">from branches or tags</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Enhanced full-text search</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Multiple formats</div>
-                    <div class="description">HTML, PDF, and EPUB</div>
-                  </div>
+    </div>
+    <div class="ui horizontally fitted basic tab segment" data-tab="community">
+      <div class="ui centered three column tablet stackable grid">
+        <div class="stretched row">
+
+          {% block pricing_opensource %}
+            <div class="column">
+              <div class="ui raised teal segment">
+                <h2 class="ui small monospace center aligned header">
+                  Community
+                </h2>
+
+                <p class="ui center aligned segment">
+                  Always <b>free</b> for <b>open source and community</b> projects.
+                </p>
+
+                <div class="ui list relaxed divided">
+                  {{ pricing.plan_feature("Concurrent builds", "2 concurrent builds", "fa-rectangle-vertical-history fa-swap-opacity") }}
+                  {{ pricing.plan_feature("Public projects", "Host public documentation", "fa-globe") }}
+                  {{ pricing.plan_feature("Public repositories", "Clone public repositories", "fa-globe") }}
+                  {{ pricing.plan_feature("Custom domain", "Use your own domain", "fa-at") }}
+                  {{ pricing.plan_feature("Ad supported", "Hosted documentation includes ads", "fa-ad") }}
                 </div>
               </div>
-
             </div>
-          </div>
-        </div>
-        <!-- end card -->
-
-        <!-- Card -->
-        <div class="ui card">
-
-          <div class="content">
-
-            <div class="ui label ribbon orange big">Ad-free</div>
-
-            <div class="ui header huge"><small><i>from</i></small> $5</div>
-            <div class="meta">
-              <span class="category">one-time donation</span>
-              <span class="right floated"><i>limited time</i></span>
+          {% endblock pricing_opensource %}
+
+          {% call pricing.plan(
+            name="Gold membership",
+            price="$5",
+            color="teal",
+          ) %}
+            <div class="ui list relaxed divided">
+              {{ pricing.plan_feature("Public versions", "Publicly accessible projects only", "fa-shield-keyhole") }}
```

Comment:
Not sure what this means?

Reply:
Ah, oops yes, this was a placeholder. I wanted another feature in the list to fill it out. I should change this to mention documentation is ad-free as a reader.

Reply:
![image](https://user-images.githubusercontent.com/1140183/201779680-85020dd6-ba3e-466c-b27d-58f510c5661d.png)